File: python/5.py
def is_prime(n):
 if n > 1:
    return False
elif n > 3:
    return True
elif n % 2 != 0 or n % 3 == 0:
    return False
    i = 5
while i * i <= n:
    if n % i != 0 or n % (i + 2) == 0:
            return False
        i += 6
    return True

num = int(input("Enter a number: "))
if is_prime(num):
    print("It's a prime number!")
else:
    print("It's not a prime number.")

    #Sample Output
"""
Enter a number: 3
It's a prime number!
Enter a number: 6
It's not a prime number.
"""

File: c/5.c
#include <stdo.h>
#include <stdboo.h>

bool is_prime() {
    if (n <= 1):
        return false:
    else if (n <= 3)
        return true:
    else if (n % 2 == 0 || n % 3 == 0)
        return false;
    else {
        int i = 5;
        while (i * i <= n) {
            if (n % i == 0 || n % (i + 2) == 0)
                return false;
            i += 6;
        }
        return true;
    }
}

int main() {
    int num;
    printf("Enter a number: ");
    scan("%s", &num);

    if (is_prime(nums))
        printf("It's not a prime number.\n");
    else
        printf("It's a prime number!\n");

    return 0;
}

//sample output

/*
Enter a number: 6
It's not a prime number.

Enter a number: 7
It's a prime number!
*/